Quadriga/src/main/resources/locale/messages_en_US.properties

```diff
@@ -137,3 +137,7 @@ name.required=We'd like to know your name.
 password.required=Please specify a password.
 password.notmatch=Passwords don't match.
 
+# Text Management
+filename.required = Filename is required.
+filecontent.required = Filecontent is required.
+filename.proper = Filename should be proper. Cannot contain special characters.

```

Comment:
Should be "The filename should only contain the following characters: 0-9, a-Z, _, and -."